web/src/main/java/org/perfrepo/web/service/UserServiceBean.java

```diff
@@ -256,11 +259,13 @@ public boolean isLoggedUserInGroup(String guid) {
 
 	@Override
 	public boolean isUserInGroup(Long userId, Long groupId) {
-		User user = getUser(userId);
-		if (user != null && user.getGroups() != null) {
-			for (Group group : user.getGroups()) {
-				if (groupId.equals(group.getId())) {
-					return true;
+		if (userId != null) {
+			User user = getUser(userId);
+			if (user != null && user.getGroups() != null) {
+				for (Group group : user.getGroups()) {
+					if (groupId.equals(group.getId())) {
+						return true;
+					}
```

Comment:
IMHO, this syntax is much more readable, let's leverage Java 8! :)

@Override
	public boolean isUserInGroup(Long userId, Long groupId) {
		if (userId == null) {
			return false;
		}

		User user = getUser(userId);
		if (user != null && user.getGroups() != null) {
			return user.getGroups().stream().anyMatch(group -> groupId.equals(group.getId()));
		}

		return false;
	}